packages/block-editor/src/hooks/auto-inserting-blocks.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { addFilter } from '@wordpress/hooks';
+import { Fragment } from '@wordpress/element';
+import { PanelBody, ToggleControl } from '@wordpress/components';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { createBlock, store as blocksStore } from '@wordpress/blocks';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { InspectorControls } from '../components';
+import { store as blockEditorStore } from '../store';
+
+function AutoInsertingBlocksControl( props ) {
+	const { autoInsertedBlocksForCurrentBlock, groupedAutoInsertedBlocks } =
+		useSelect(
+			( select ) => {
+				const { getBlockTypes } = select( blocksStore );
+				const _autoInsertedBlocksForCurrentBlock =
+					getBlockTypes()?.filter(
+						( block ) =>
+							block.autoInsert &&
+							Object.keys( block.autoInsert ).includes(
+								props.blockName
+							)
```

Comment:
looks like we can use `in` here again to simplify the code and allocate fewer arrays. maybe this got lost in a rebase because I thought you included it (pun intended)

```js
getBlockTypes()?.filter( ( { autoInsert } ) => autoInsert && props.blockName in autoInsert );
```

Reply:
I replaced another occurrence but did miss this one!